resources/views/themes/bimbala/partials/footer.blade.php

```diff
@@ -20,14 +20,14 @@
                         <li class="box-border py-2 text-sm font-normal text-left text-gray-500 md:text-base md:mb-1">
                             <a href="{{ route('try-it') }}" class="text-base leading-tight no-underline bg-transparent cursor-pointer hover:text-gray-400">Try it</a>
                         </li>
-                        {{--  <li class="box-border py-2 text-sm font-normal text-left text-gray-500 md:text-base md:mb-1">
-                            <a href="#_" class="text-base leading-tight no-underline bg-transparent cursor-pointer hover:text-gray-400">Changelog</a>
+                        {{-- <li class="box-border py-2 text-sm font-normal text-left text-gray-500 md:text-base md:mb-1">
+                            <a href="{{ route('use-case') }}" class="text-base leading-tight no-underline bg-transparent cursor-pointer hover:text-gray-400">Use Case</a>
```

Comment:
Is this change really necessary?

Reply:
My idea was to place it in the correct "category" or at least to use the categories which we have made for now. I changed the "old" one because it will be used for something else. I'm open for discussion if necessary.

Reply:
I see.